servatrice/migrations/servatrice_0020_to_0021.sql

```diff
@@ -0,0 +1,12 @@
+
+
+CREATE TABLE IF NOT EXISTS `cockatrice_forgot_password` (
+  `id` int(7) unsigned zerofill NOT NULL auto_increment,
+  `name` varchar(35) NOT NULL,
+  `requestDate` datetime NOT NULL default '0000-00-00 00:00:00',
```

Comment:
I can add the two, though I had intended on creating an audit PR next that would record that information.  Since the functionality was designed to mimic the registration process and it doesn't record any of that information I didnt include it in the forgot password.